lib/matplotlib/backend_bases.py

```diff
@@ -690,7 +694,7 @@ def __init__(self):
         self._linewidth = 1
         self._rgb = (0.0, 0.0, 0.0, 1.0)
         self._hatch = None
-        self._hatch_color = colors.to_rgba(rcParams['hatch.color'])
+        self._hatch_color = (0.0, 0.0, 0.0, 1.0)
```

Comment:
The `hatch.color` rcparam is changed to `inherit`. The logic for using the value in rcParam is added within the `set_hatchcolor()` functions of patches and collections. The (0.0, 0.0, 0.0, 1.0) value here is just a default, which never actually gets used. This is similar to how other variables such as `_rgb` and `_linewidth` having a default value.

Reply:
I don't think we can change the default `hatch.color` to `'inherit'`. 

If we are going to put a "never used" place holder here we should use `None` rather than a valid color?